jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/Users.java

```diff
@@ -2,6 +2,7 @@
 
 import io.github.com.entities.User;
 
+@SuppressWarnings("PMD.ClassNamingConventions")
 public class Users {
```

Comment:
This class looks similar to Users(Utils) above. I think we should renamed it too